x/market/internal/keeper/swap.go

```diff
@@ -0,0 +1,151 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	core "github.com/terra-project/core/types"
+	"github.com/terra-project/core/x/market/internal/types"
+)
+
+// UpdateDelta updates each pool's delta
```

Comment:
Explanation is recursive; consider adding explanation regarding what the delta is (A - B, specify A and B)

Reply:
Done